db/migrate/20200317203456_create_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEvents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :events do |t|
+      t.datetime :event_date
+      t.string :location
+      t.string :description
+      t.references :user, null: false, foreign_key: true
```

Comment:
**[REQUIRED]**
* events table has creator_id as the foreign key ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))